.github/workflows/2_bump_versions.yml

```diff
@@ -0,0 +1,55 @@
+name: Bump versions
+
+on:
+  workflow_dispatch:
+    inputs:
+      npm_channel:
+        description: 'Prisma CLI NPM Channel'
+        required: true
+      version:
+        description: 'New Prisma CLI version'
+        required: true
+
+env:
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+  ENVIRONMENT: ${{ secrets.ENVIRONMENT }}
+
+jobs:
+  bump:
+    runs-on: ubuntu-latest
+    steps: 
+      - uses: actions/checkout@v2
+      - name: Install Semver # Needed to use semver in scripts
+        run: npm install
+      - name: Setup Git
+        run: |
+          git config --global user.email "prismabots@gmail.com"
+          git config --global user.name "Prismo"
+      - name: Update versions in scripts folder 
+        run: npm run bump_prisma_version_in_scripts ${{ github.event.inputs.npm_channel }} ${{ github.event.inputs.version }}
```

Comment:
As we have both the channel and the version here - can we maybe get rid of the script completely? This could just be a one liner script here if we do not need the additional logging (and we rename the filename of `patch-dev` as suggested in another comment). Or is this used in multiple places that justifies this extraction?

Reply:
Yes, we can get rid of it, I replaced it with a one-line shell command